components/progressbar/index.css

```diff
@@ -11,40 +11,84 @@ governing permissions and limitations under the License.
 */
 
 @import "../commons/index.css";
+@import "../vars/css/components/spectrum-progressbar.css";
+@import "../vars/css/components/spectrum-meter.css";
+@import "../vars/css/components/spectrum-fieldlabel.css";
+
+.spectrum-ProgressBar--sizeS {
+  @remapvars {
+    find: --spectrum-progressbar-s-;
+    replace: --spectrum-progressbar-;
+  }
+  @remapvars {
+    find: --spectrum-fieldlabel-side-s-;
+    replace: --spectrum-fieldlabel-side-;
+  }
+}
+
+.spectrum-ProgressBar--sizeM {
+  @remapvars {
+    find: --spectrum-progressbar-m-;
+    replace: --spectrum-progressbar-;
+  }
+  @remapvars {
+    find: --spectrum-fieldlabel-side-m-;
+    replace: --spectrum-fieldlabel-side-;
+  }
+}
+
+.spectrum-ProgressBar--sizeL {
+  @remapvars {
+    find: --spectrum-progressbar-l-;
+    replace: --spectrum-progressbar-;
+  }
+  @remapvars {
+    find: --spectrum-fieldlabel-side-l-;
+    replace: --spectrum-fieldlabel-side-;
+  }
+}
+
+.spectrum-ProgressBar--sizeXL {
+  @remapvars {
+    find: --spectrum-progressbar-xl-;
+    replace: --spectrum-progressbar-;
+  }
+  @remapvars {
+    find: --spectrum-fieldlabel-side-xl-;
+    replace: --spectrum-fieldlabel-side-;
+  }
+}
 
 .spectrum-ProgressBar {
   position: relative;
   display: inline-flex;
   flex-flow: row wrap;
   justify-content: space-between;
   align-items: center;
-  inline-size: var(--spectrum-progressbar-large-width);
+  inline-size: var(--spectrum-progressbar-width);
```

Comment:
>  It might be better for the default values to references the correct DNA-var and not a custom abbreviation. Finding & comparing the values in the DNA compiled output would be easier as there is currently no default spectrum-progressbar-width token.

We are using `@remapvars` to turn sized variables into unsized variables and apply them. There is no usage of `--spectrum-progressbar-m-width` because the mechanism by which we implement t-shirt sizing is changing the value of `--spectrum-progressbar-width` to equal either `--spectrum-progressbar-m-width` or `--spectrum-progressbar-l-width` etc. This was reviewed and accepted in #1044 and it's far too late to change it, if that's what you're implying.

>I think are are working here on a default progress-bar classname that can be applied without specifying t-shirt sizing.

No, we discussed applying default size at length and we want to require a size class to be applied. [You +1'd it haha](https://adobedesign.slack.com/archives/G019JTYMT6H/p1602005622020900?thread_ts=1601998888.004300&cid=G019JTYMT6H)